src/world/Network/PacketWrappers/ExaminePacket.h

```diff
@@ -49,6 +49,8 @@ namespace Sapphire::Network::Packets::Server
         m_data.mainWeaponModel = pTarget->getModelMainWeapon();
         m_data.secWeaponModel = pTarget->getModelSubWeapon();
 
+        m_data.worldId = 67; // TODO: world id from server
```

Comment:
You should use the world ID variable from general config for these

Reply:
that won't really work unless you pull framework in which doesn't feel like a good solution in the case of packet wrappers

Reply:
We should probably just store the current server in the player.